.test-infra/jenkins/job_LoadTests_SideInput_Flink_Go.groovy

```diff
@@ -0,0 +1,99 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonTestProperties
+import CommonJobProperties as commonJobProperties
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+import InfluxDBCredentialsHelper
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def batchScenarios = {
+  [
+    [
+      title          : 'SideInput Go Load test: 10gb-1kb-10workers-1window-first-iterable',
+      test           : 'sideinput',
+      runner         : CommonTestProperties.Runner.FLINK,
+      pipelineOptions: [
+        job_name           : "load-tests-go-flink-batch-sideinput-1-${now}",
+        influx_namespace   : 'flink',
+        influx_measurement : 'go_batch_sideinput_1',
+        input_options      : '\'{' +
+        '"num_records": 10000000,' +
+        '"key_size": 100,' +
+        '"value_size": 900}\'',
+        parallelism        : 10,
+        endpoint           : 'localhost:8099',
+        environment_type   : 'DOCKER',
+        environment_config : "${DOCKER_CONTAINER_REGISTRY}/beam_go_sdk:latest",
```

Comment:
This test is exactly the same as one below. Did you miss `access_elements`?

Reply:
@kamilwu if you look at `job_LoadTests_SideInput_Python.groovy` there are two iterable tests. They are exactly the same with the exception of a name, here's the fragment of code:
```groovy
    [
      name: '10gb-1kb-10workers-1window-first-iterable',
      testSpecificOptions: [
        input_options    : '\'{' +
        '"num_records": 10000000,' +
        '"key_size": 100,' +
        '"value_size": 900}\'',
        side_input_type  : 'iter',
        access_percentage: 1,
      ]
    ],
    [
      name: '10gb-1kb-10workers-1window-iterable',
      testSpecificOptions: [
        input_options    : '\'{' +
        '"num_records": 10000000,' +
        '"key_size": 100,' +
        '"value_size": 900}\'',
        side_input_type  : 'iter',
      ]
    ],
```
And default access_percentage is 1, and it's not defined in second test case.

Reply:
@kamilwu  Sorry, you're right default access_percentage is 100, therefore first test should have access_percentage 1, and second one none, which means it gets the default 100.